db/migrate/20211126190017_create_reservation.rb

```diff
@@ -0,0 +1,14 @@
+class CreateReservation < ActiveRecord::Migration[6.1]
+  def change
+    create_table :reservations do |t|
+      t.references :branch
+      t.references :customer
+      t.integer :person_count
+      t.datetime :booking_schedule
+      t.integer :tables_booked, default: 0
+      t.integer :total_amount, default: 0
```

Comment:
never give default value 0 to total_amount. might result in free reservations being created if there is a bug in the code

Reply:
you can give 0 defaults to fields like discounts, which can be left 0 easily

Reply:
Ok Sir.
I was directly adding amount for line_items created in code, so set 0 default.
Will remember this.

Reply:
Fixed.